src/services/services.js

```diff
@@ -0,0 +1,67 @@
+import firebase, { db } from './firebase';
+
+export const isCorrectEmail = (email) => {
+  // eslint-disable-next-line no-useless-escape
+  const regExp = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  return regExp.test(email);
+};
+
+export const isCorrectPassword = (password) => {
+  // eslint-disable-next-line no-useless-escape
+  if (password.length < 8) {
+    return false;
+  }
+  return true;
+};
+
+export const loginFirebase = async (email, password) => {
+  try {
+    const response = await firebase.auth().signInWithEmailAndPassword(email, password);
+    return response;
+  } catch (error) {
+    throw new Error('Check your login and password');
+  }
+};
+
+export const registerNewUserFirebase = async (email, password) => {
+  try {
+    const response = await firebase.auth().createUserWithEmailAndPassword(email, password);
+    return response;
+  } catch (error) {
+    throw new Error('An error occured :(');
+  }
+};
+
+export const logoutFirebase = async () => {
+  try {
+    await firebase.auth().signOut();
+  } catch (error) {
+    throw new Error('An error occured during logout. Try later.');
+  }
+};
+
+export const getDataFirebase = async (collection, field) => {
```

Comment:
Don't need to mark method as `async` if you do not use `await`

Reply:
thx, i changed this.
```suggestion
    export const getUserStatus = async () => {
  await firebase.auth().onAuthStateChanged((user) => Boolean(user));
};
```